components/webpack/webpack.config.js

```diff
@@ -17,7 +17,13 @@ module.exports = (env, argv) => ({
   resolve: {
     extensions: ['.js', '.tsx', '.ts', '.json'],
     alias: {
+      // Find files in the current build configurations /gen directory
+      'gen': process.env.ROOT_GEN_DIR,
```

Comment:
Storybook needs to know about the `gen` directory as well

Reply:
For now I've told it about a `gen/` folder that we can _mock_. Before relegating storybook to being something that requires the browser build before it can be built, I'm going to attempt to copy snapshots of mojom-generated Types to this directory. Since no stories in this PR actually need anything from `gen/`, I'm going to have to do that in another branch when this first comes up (probably Brave News PR).